website/docs/providers/entra/add-entra-provider.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Add an Entra ID provider
+---
+
+<span class="badge badge--primary">Enterprise</span>
+
+---
+
+For more information about using a Entra ID provider, see the [Overview](./index.md) documentation.
+
+:::info
+This feature is in technical preview, so please report any bugs on [GitHub](https://github.com/goauthentik/authentik/issues).
+:::
+
+## Prerequisites
+
+To create a Entra ID provider provider in authentik, you must have already [configured Entra ID](./setup-entra.md) to integrate with authentik. You will need to obtain from Entra the Application (client) ID, the Directory (tenant) ID, and the Client secret values, to enter into authentik when adding an Entra ID provider.
```

Comment:
Typo? "You will need to obtain from Entra ID..."

Reply:
No typo but I reworded it to be less confusing.